gitgud/levels/rampup/_relrefs1/instructions.txt

```diff
@@ -0,0 +1,15 @@
+Now that we've learned how to detach the HEAD, let's learn easier ways to refer to a commit.
+>>>
+In the previous exercise, we learned that we only need 4-digits of a commit hash in order to reference a particular commit. This method made the process much easier than entering in a giant string of characters and numbers (e.g. fed2da64c0efc5293610bdd892f82a58e8cbc5d8). However, in this exercise we'll learn even more ways of refering to commits!
+>>>
+In particular there are two ways of referencing commits called "relative refs." Relative refs are used to refer to a commit nodes that are a couple commits behind your current commit. It should be noted that using relative refs are only efficient to use if the commit you are referring is nearby in the tree; otherwise, in many cases it may be faster to bring up the commit log and use the hash.
```

Comment:
Can you call them "relative references" when you introduce them just to be clear? After `called "relative references"`, I think it's fine to call them refs, people should be able to pick up on that